reference/tax_provider.yml

```diff
@@ -641,6 +653,7 @@ paths:
                         shipping:
                           id: string
                           item_code: string
+                          item_reference: '12345678'
```

Comment:
🍹 Is it worth keeping this in line with item_code and having `string`?
Also curious about the examples being a mix of 'string' and string types.

Reply:
@benpratt77 these are automatically generated, for the most part.